### PR TITLE
Vb/chore/change selected row background in table

### PR DESCRIPTION
**Pull request by vadimbasko**

### CODE REVIEW

File: example-app/src/app/screens/owner/OwnerTableEditor.tsx [hunk 1]
```diff
@@ -21,26 +21,30 @@ import { FormattedMessage, useIntl } from "react-intl";
 import { gql2form } from "../../../core/format/gql2form";
 import { RefetchQueries } from "../../../core/type-aliases/RefetchQueries";
 
-const OWNER = gql(/* GraphQL */ `
```
Comment (vyacheslav-pushkin) [inline example-app/src/app/screens/owner/OwnerTableEditor.tsx:24]: Please rebase on latest master

**Review by vyacheslav-pushkin — CHANGES_REQUESTED** [review]
Please rebase and merge

File: example-app/src/app/screens/owner/OwnerTableEditor.tsx (hunk 1 above)
Comment (vadimbasko) [reply example-app/src/app/screens/owner/OwnerTableEditor.tsx]: fixed

**Comment by vadimbasko** [comment]
All requested changes fixed. Still can't merge - 'merging is blocked'. @vyacheslav-pushkin please approve again.

**Pull request merged**